private/api/classrooms/get-classroom.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require_once('../db-connection/root-connection.php');
+try{
+
+    if(isset($_GET['classroom_id'])){
+        //GET ID
+        $classroom_id = $_GET['classroom_id'];
+        //Prepare SQL query
+        $q = $db->prepare('SELECT * FROM classrooms WHERE id = :classroom_id');
```

Comment:
classroom_id in SQL . Should it be classroomID ? Lets discuss :)

Reply:
Skulle vi ikke følge syntaksen vi havde lavet? <3